elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -274,6 +279,37 @@ defmodule API.Client.Channel do
     end
   end
 
+  def handle_info(
+        %Phoenix.Socket.Broadcast{
+          event: "presence_diff",
+          topic: "presences:relays:" <> relay_id,
+          payload: %{leaves: leaves}
+        },
+        socket
+      ) do
+    OpenTelemetry.Ctx.attach(socket.assigns.opentelemetry_ctx)
+    OpenTelemetry.Tracer.set_current_span(socket.assigns.opentelemetry_span_ctx)
+
+    if Map.has_key?(leaves, relay_id) do
+      OpenTelemetry.Tracer.with_span "client.relay_offline",
+        attributes: %{
+          relay_id: relay_id
+        } do
+        {:ok, relays} = select_relays(socket)
+        :ok = Enum.each(relays, &Domain.Relays.subscribe_to_relay_presence/1)
+
+        push(socket, "relay_offline", %{
```

Comment:
I figured we could call this `relay(s)_status_update` or something similar. At least from how the data structure is defined, `offline_relay_ids` isn't necessarily set so we could also use this to just broadcast the presence of new relays.

Overall, I think we should name these events after what they describe and not when they are emitted.

Reply:
@thomaseizinger why would we broadcast new relays, if we don't take into account RTT there is no point to do connection upgrades, right?

Reply:
Who knows what we'll do in the future :)

The point I was trying to make is: Naming events / messages after what they describe is IMO a more stable identifier compared to naming after _when_ they are emitted. The latter might change as we evolve the software.

Here, we are notifying the client about a status change of several relays: Some went offline, some went online. As the client, I don't care when you are sending me this, I'll always process it the same way.

Also, `relay_offline` is currently singular but it has a list of IDs. At the least, we should call it `relays_offline`.

Reply:
Ah sorry @thomaseizinger for missing your message. Naming is always hard, wdyt about renaming it to “relays_presence” and have two maps: disconnected_ids: [uuid], and connected: [relay, ..]

Reply:
> Ah sorry @thomaseizinger for missing your message. Naming is always hard, wdyt about renaming it to “relays_presence” and have two maps: disconnected_ids: [uuid], and connected: [relay, ..]

Yeah that works for me!